aplus/settings.py

```diff
@@ -71,14 +71,22 @@
 # local_templates/possible_path/template_name.html
 ##########################################################################
 SITEWIDE_ALERT_TEXT = None
-SITEWIDE_ADVERT = None
+SITEWIDE_ADVERT = {
+   'not-before': '2025-01-01', # start showing on 1st
+   'not-after': '2026-01-04', # last visible date is 3rd
+   'title': {'en': "Advert", 'fi': "Mainos"},
+   'text': {'en': "We have open positions",
+            'fi': "Meillä on paikkoja"},
+   'href': "https://apluslms.github.io",
+   'image': "https://apluslms.github.io/assets/images/apluslogo.png",
+}
```

Comment:
This should be removed after it has been used to verify that the advert looks good.